src/taipy/config/_config.py

```diff
@@ -25,6 +25,14 @@ def __init__(self):
         self._unique_sections: Dict[str, UniqueSection] = {}
         self._global_config: GlobalAppConfig = GlobalAppConfig()
 
+    def clear(self):
+        self._global_config.clear()
+        for unique_section in self._unique_sections.values():
+            unique_section.clear()
+        for sections in self._sections.values():
+            for section in sections.values():
+                section.clear()
```

Comment:
I prefer the name `clean` as it does not make the _config empty, but it cleans all the elements.
Plus, can you make it hidden?

```suggestion
    def _clean(self):
        self._global_config._clean()
        for unique_section in self._unique_sections.values():
            unique_section._clean()
        for sections in self._sections.values():
            for section in sections.values():
                section._clean()
```